django/core/signing.py

```diff
@@ -72,10 +74,16 @@ def base64_hmac(salt, value, key):
     return b64_encode(salted_hmac(salt, value, key).digest()).decode()
 
 
+def _cookie_signer_key(key):
+    return b'django.http.cookies' + force_bytes(key)
```

Comment:
Let's carry over the comment.

```suggestion
    return b'django.http.cookies' + force_bytes(key)  # SECRET_KEY may be str or bytes.
```